WordPress/src/main/java/org/wordpress/android/ui/mysite/ImprovedMySiteFragment.kt

```diff
@@ -173,7 +173,9 @@ class ImprovedMySiteFragment : Fragment(),
             it?.let { uiModel ->
                 loadGravatar(uiModel.accountAvatarUrl)
                 when (val state = uiModel.state) {
-                    is State.SiteSelected -> loadData(state.items)
+                    is State.SiteSelected -> {
+                        loadData(state.items)
+                    }
```

Comment:
Big nitpick here, just to maintain consistency.

```suggestion
                    is State.SiteSelected -> loadData(state.items)
```